.gitbook/FAQ.md

```diff
@@ -185,7 +185,7 @@ When you make an Ethereum transaction, the user will be given an expected fee fo
 
 Here, Boba is looking out for you. Before that unexpected increased fee goes through, Boba throws an error message and prevents the payment from going through. If the transaction fee is at least three times what the expected cost was (in this example, your transaction fee of 3 USD jumped to 9 USD), the transaction will fail and saves you from the increased fee.
 
-[Here's more info on fees](https://docs.boba.network/developer-docs/fee-scheme#for-frontend-and-wallet-developers).
+[Here's more info on fees](https://docs.boba.network/developer/fee-scheme#for-frontend-and-wallet-developers).
```

Comment:
Again, probably out of scope, but this anchor doesn't seem to exist.

Reply:
Fixed